frontend/pages/admin/OrgSettingsPage/cards/Advanced/Advanced.tsx

```diff
@@ -72,12 +109,13 @@ const Advanced = ({
         enable_analytics: appConfig.server_settings.enable_analytics,
         query_reports_disabled: disableQueryReports,
         scripts_disabled: disableScripts,
+        deferred_save_host: appConfig.server_settings.deferred_save_host,
       },
       smtp_settings: {
         enable_smtp: appConfig.smtp_settings?.enable_smtp || false,
         sender_address: appConfig.smtp_settings?.sender_address || "",
         server: appConfig.smtp_settings?.server || "",
-        port: Number(appConfig.smtp_settings?.port),
+        port: appConfig.smtp_settings?.port || undefined,
```

Comment:
`appConfig.smtp_settings?.port` will already be `undefined` if it's not a `number` so no need to `|| undefined`